packages/RichText-Core.package/RTETextComponent.class/instance/keyStroke..st

```diff
@@ -0,0 +1,5 @@
+event handling
+keyStroke: anEvent
+	
+	self coreObject addAttribute: (TextFontReference toFont: (TTCFont familyName: 'Merriweather' pointSize: 16  emphasis: 0)).
+	"1 to: 100 do: [:i | self coreObject addAttribute:(TextStyle new addNewFontSize: i)]."
```

Comment:
what is this loop doing? unused code should be deleted.

Reply:
wanted to use it later to be able to input any number in fontsizefield and actually change the font size to that number. isn't working right know so can also delete for now